networking/build.gradle

```diff
@@ -10,26 +10,34 @@ android {
     defaultConfig {
         minSdkVersion 16
         targetSdkVersion 25
-        consumerProguardFiles 'proguard-joda-time.pro', 'proguard-retrofit.pro',
-                'proguard-okhttp.pro'
+        consumerProguardFiles 'proguard-retrofit.pro', 'proguard-okhttp.pro', 'proguard-gson.pro'
     }
 
     lintOptions {
         abortOnError false
     }
 }
 
+buildscript {
+    ext.dagger_version = '2.11-rc2'
+    ext.retrofit_version = '2.2.0'
+    ext.okhttp_version = '3.7.0'
+}
+
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
 
     // Wolmo
-    compile 'com.github.Wolox:wolmo-core-android:v1.0.0'
+    compile 'com.github.Wolox:wolmo-core-android:dagger-android-SNAPSHOT'
```

Comment:
We should change the `dagger-android-SNAPSHOT` to the corresponding version when merging the PR. Maybe put a `TODO` :thinking:?

Reply:
Nice catch, I'll add it :smile: